auth/index.html

```diff
@@ -233,9 +233,8 @@ <h2>Message log</h2>
         /**
          * Removes the embedded key from local storage.
          */
-        var onResetEmbeddedKey = function () {
+        var resetEmbeddedKey = function () {
```

Comment:
For readers: functions named `on{action}` differ from those named `{action}` in that the latter are the the main utility functions, and the former are handlers.

So `onResetEmbeddedKey()` gets called when an inbound message is received (i.e. from the parent), and `resetEmbeddedKey()` is responsible for the main guts of the logic (i.e. clearing the embedded key from local storage)